### PR TITLE
Update weather-bot cookbook

### PR DESCRIPTION
**Pull request by jakubkornafel**

### CODE REVIEW

File: docs/cookbook/web/solutions/weather-bot.md [hunk 1]
```diff
@@ -103,43 +107,52 @@ event_handlers:
...
-       response.push(`${prediction.hour} - ${prediction.forecast}`)
-       if (prediction.forecast.toLowerCase() == 'rain') {
-         rain = true
-       }
+import Syncano from '@syncano/core'
+
+export default async ctx => {
+  const {event, endpoint} = new Syncano(ctx)
+  const {text, sender} = ctx.args
+  console.log(text)
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/weather-bot.md]: Let's remove console.log, if you need logging use logger.

File: docs/cookbook/web/solutions/weather-bot.md [hunk 2]
```diff
@@ -103,43 +107,52 @@ event_handlers:
...
+
+  if (text.toLowerCase().indexOf('hello') >= 0) {
+    event.emit('messenger-bot.message-send', {text: `Hi, this is Syncano Weather bot 😎 
+To check the weather, please type your city name in your message ie. Oslo  `, sender} )
+  } else if (text.toLowerCase().indexOf('help') >= 0) {
+    event.emit('messenger-bot.message-send', {text: `It looks like you need some help 🤔 
+To check the weather, please type your city name in your message ie. Oslo  `, sender} )
+  } else {
+    endpoint.post('openweathermap/get-three-hours', {city: text})
+    .then(forecast => {
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/weather-bot.md]: PLease refactor to async/await.

File: docs/cookbook/web/solutions/weather-bot.md [hunk 3]
```diff
@@ -78,7 +82,7 @@ Choose empty project for this Socket:
 
 #### Building logic of the "responder" Socket
 
-First thing is to catch en event about received message, to do that we have to add event handler o to the Socket config file (syncnao/responder/socket.yml):
+First thing is to catch an event about received message, to do that we have to add event handler to the Socket config file (syncano/responder/socket.yml):
 
 ```yaml
 event_handlers:
```
Comment (maciejkorsan) [inline docs/cookbook/web/solutions/weather-bot.md]: this is an old yaml file

File: docs/cookbook/web/solutions/weather-bot.md [hunk 4]
```diff
@@ -51,7 +51,11 @@ Then you have to provide `Page Token`, follow the instructions to get it from Fa
     To find the token, go to the 'Messenger > Settings > Token Generation' section in your Facebook Application settings panel at https://developers.facebook.com.
 
     Type in value: EAAbvRMZClZC4YBAL63OHJLSMWwbSKg9BM1eojt2VU0fv95vkgURMjqDqKPUVZCuN3HjNE8fjt2TJfK8Jt68fwVAAltb8JnQjgpcbMHF9eqh2OiH4ZC0ftsJz3h5ZA7wKOWacDOQGte9b9Lhl3KKuvdrJJhIgjZAAeXKgUXmSkEdgZDZD
-    Your socket is ready to use! Type npx s socket list messenger-bot to see docs.
+```
+
+Your socket is ready to use! To see its docs type:
+```sh
+ npx s list messenger-bot
 ```
```
Comment (maciejkorsan) [inline docs/cookbook/web/solutions/weather-bot.md]: missing part - webhook configuration & verification

Comment (maciejkorsan) [reply docs/cookbook/web/solutions/weather-bot.md]: there is a hub verify token phrase which is: 'messenger-bot' - you need to configure it inside facebook app settings

File: docs/cookbook/web/solutions/weather-bot.md (hunk 3 above)
Comment (maciejkorsan) [reply docs/cookbook/web/solutions/weather-bot.md]: use code from: https://github.com/eyedea-io/syncano-weather-bot-example/tree/feat/refactor this the updated version of example repository

File: docs/cookbook/web/solutions/weather-bot.md [hunk 5]
```diff
@@ -3,27 +3,32 @@
 - Preparation: **6 minutes**
 - Requirements:
   - Initiated Syncano project
-  - OpenWeatherMap API key
+  - OpenWeatherMap API
```
Comment (maciejkorsan) [inline docs/cookbook/web/solutions/weather-bot.md]: I think key was the right word here - you need a key / access to the OpenWeatherMap API :)

File: docs/cookbook/web/solutions/weather-bot.md [hunk 6]
```diff
@@ -3,27 +3,32 @@
...
 
 ### Problem to solve
 
-You want to build simple Messenger bot for Facebook page which can give you forecast for next 3 hours for particular city.
+You want to build simple Messenger bot for a Facebook page which can give you forecast for the next 3 hours for a particular city.
 
 ### Solution
 
-Our solution is based on two Sockets. [openweathermap](https://syncano.io/#/sockets/openweathermap) Socket will be used to get weather data from [OpenWeatherMap](https://openweathermap.org/) service using their API. [messenger-bot](https://syncano.io/#/sockets/messenger-bot) will be used for communication with Facebook. On top of that, we will create `responder` Socket which will connect those two functionalities.
+Our solution is based on two Sockets. [openweathermap](https://syncano.io/#/sockets/openweathermap) Socket will be used to get weather data from [OpenWeatherMap](https://openweathermap.org/) service using their API. [messenger-bot](https://syncano.io/#/sockets/messenger-bot) will be used for communication with Facebook. On top of that, we will create a `responder` Socket which will connect those two functionalities.
```
Comment (maciejkorsan) [inline docs/cookbook/web/solutions/weather-bot.md]: old links - you should give npm / github links to the sockets here

File: docs/cookbook/web/solutions/weather-bot.md [hunk 7]
```diff
@@ -51,11 +64,7 @@ Then you have to provide `Page Token`, follow the instructions to get it from Fa
     To find the token, go to the 'Messenger > Settings > Token Generation' section in your Facebook Application settings panel at https://developers.facebook.com.
 
     Type in value: EAAbvRMZClZC4YBAL63OHJLSMWwbSKg9BM1eojt2VU0fv95vkgURMjqDqKPUVZCuN3HjNE8fjt2TJfK8Jt68fwVAAltb8JnQjgpcbMHF9eqh2OiH4ZC0ftsJz3h5ZA7wKOWacDOQGte9b9Lhl3KKuvdrJJhIgjZAAeXKgUXmSkEdgZDZD
```
Comment (maciejkorsan) [inline docs/cookbook/web/solutions/weather-bot.md]: Is this a real key?

File: docs/cookbook/web/solutions/weather-bot.md [hunk 8]
```diff
@@ -51,11 +64,7 @@ Then you have to provide `Page Token`, follow the instructions to get it from Fa
...
-```
-
-Your socket is ready to use! To see its docs type:
-```sh
- npx s list messenger-bot
+    Your socket is ready to use! Type `npx s socket list` messenger-bot to see docs.
```
Comment (maciejkorsan) [inline docs/cookbook/web/solutions/weather-bot.md]: does it really shows the docs? or list of endpoints

File: docs/cookbook/web/solutions/weather-bot.md (hunk 7 above)
Comment (jakubkornafel) [reply docs/cookbook/web/solutions/weather-bot.md]: No. It was in the old version of cookbook.

File: docs/cookbook/web/solutions/weather-bot.md (hunk 8 above)
Comment (jakubkornafel) [reply docs/cookbook/web/solutions/weather-bot.md]: It shows list of endpoints with their additional information. Should I change it to: 
" Type `npx s socket list` messenger-bot to see further information."

File: docs/cookbook/web/solutions/weather-bot.md (hunk 5 above)
Comment (jakubkornafel) [reply docs/cookbook/web/solutions/weather-bot.md]: The "key" word is back.

File: docs/cookbook/web/solutions/weather-bot.md (hunk 7 above)
Comment (jakubkornafel) [reply docs/cookbook/web/solutions/weather-bot.md]: I've changed the key value into text: "your-key-value" etc.

File: docs/cookbook/web/solutions/weather-bot.md (hunk 6 above)
Comment (jakubkornafel) [reply docs/cookbook/web/solutions/weather-bot.md]: Done. All Sockets have npm links.

File: docs/cookbook/web/solutions/weather-bot.md (hunk 8 above)
Comment (jakubkornafel) [reply docs/cookbook/web/solutions/weather-bot.md]: Changes have been introduced.

**Pull request merged**